kotlinx-coroutines-core/build.gradle

```diff
@@ -17,14 +20,25 @@ if (rootProject.ext.native_targets_enabled) {
 
 apply from: rootProject.file("gradle/compile-js-multiplatform.gradle")
 
+// TODO: Wasm target names have been changed since 1.9.20, remove this after migration to 1.9.20
+// https://github.com/Kotlin/kotlinx.coroutines/issues/3846
+def isNewWasmTargetEnabled = isKotlinVersionAtLeast(ext.kotlin_version, 1, 9, 20)
+if (isNewWasmTargetEnabled) {
+    apply from: rootProject.file("gradle/compile-wasm-multiplatform.gradle")
+} else {
+    apply from: rootProject.file("gradle/compile-wasm-multiplatform19.gradle")
+}
+
 apply from: rootProject.file('gradle/dokka.gradle.kts')
 apply from: rootProject.file('gradle/publish.gradle')
 /* ==========================================================================
   Configure source sets structure for kotlinx-coroutines-core:
 
      TARGETS                            SOURCE SETS
      -------         ----------------------------------------------
-
+     wasmJs -------------------------------------------------+
```

Comment:
There is `wasmJs` and `jsWasm` source sets, but diagram mentions only one of them

Reply:
Technically jsWasm is not sourceSet.